packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss

```
@@ -9,15 +9,8 @@
 		justify-content: space-between;
 		margin-bottom: $padding-small;
 
-		.left {
-			padding-left: $padding-small;
-		}
-		.right {
-			padding-right: $padding-small;
-		}
-
 		:global(.btn-icon-only) {
-			padding: 0;
+			padding: 0 0.5rem;
```

```suggestion
			padding: 0 $padding-smaller;
```

https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_paddings.scss